examples/rust/Earthfile

```diff
@@ -0,0 +1,13 @@
+FROM rust:1.24.0
+
+WORKDIR /usr/src/app
+
+build:
+    COPY . .
+    RUN cargo install
+    SAVE ARTIFACT /usr/src/app /usr/src/app
+
+docker:
+    COPY +build/usr/src/app /usr/src/app
+    CMD ["./target/release/rust-earthly"]
```

Comment:
It would be better to use `ENTRYPOINT` for this - it's not clear what the entrypoint is for the base image (if an Earthly user tries using a different base image, it might not work).